src/flag_gems/runtime/__init__.py

```diff
@@ -0,0 +1,21 @@
+from . import backend, commom_utils, register
+from .backend.device import DeviceDetector
+from .configloader import ConfigLoader
+
+configer = ConfigLoader()
+device = DeviceDetector()
+
+
+def get_triton_config(op_name):
+    return configer.get_triton_config(op_name)
+
+
+def to_register(*args, **kargs):
+    register.to_register(*args, **kargs)
```

Comment:
I didn't find the definition of to_register in register.py

Reply:
This PR has been removed, but will be used in the future